io/nvmf/nvmftest.py

```diff
@@ -20,14 +20,16 @@
 a storage networking fabric.
 """
 
-import os
-import json
 import copy
+import json
+import os
+
+import yaml
+
```

Comment:
is this line required?

Reply:
ah not require i will remove new lines which is not needed